packages/blockchain-wallet-v4/src/utils/btc.js

```diff
@@ -271,9 +272,13 @@ export const createXpubFromChildAndParent = (path, child, parent) => {
   let pathArray = bippath.fromString(path).toPathArray()
   let pkChild = compressPublicKey(Buffer.from(child.publicKey, 'hex'))
   let pkParent = compressPublicKey(Buffer.from(parent.publicKey, 'hex'))
-  let hdnode = fromPublicKey(pkChild, Buffer.from(child.chainCode, 'hex'))
-  hdnode.parentFingerprint = fingerprint(pkParent)
-  hdnode.depth = pathArray.length
-  hdnode.index = last(pathArray)
-  return hdnode.toBase58()
+  let hdnode = bip32.fromPublicKey(pkChild, Buffer.from(child.chainCode, 'hex'))
+  hdnode.__PARENT_FINGERPRINT = fingerprint(pkParent)
+  hdnode.__DEPTH = pathArray.length
+  hdnode.__INDEX = last(pathArray)
+
+  return hdnode.neutered().toBase58()
```

Comment:
Why is the `neutered()` introduced here?

Reply:
Introduced neutered() since we are not interested in the private key associated with the derived node. The derivation seems to be simply used to extract the xpub from the lockbox wallet.

This could be an unnecessary step, or perhaps I am viewing the process incorrectly.

Reply:
OK cool, if that's the case, def the right thing to do here